website/client/components/chat/chatMessages.vue

```diff
@@ -27,7 +27,7 @@
           span.action( @click='copyAsTodo(msg)')
             .svg-icon(v-html="icons.copy")
             | {{$t('copyAsTodo')}}
-          span.action(v-if='user.contributor.admin || (!msg.sent && user.flags.communityGuidelinesAccepted)', @click='report(msg)')
+          span.action(v-if='user.contributor.admin || (msg.uuid !== user._id && user.flags.communityGuidelinesAccepted)', @click='report(msg)')
```

Comment:
Then there must be something wrong with the msg.sent variable, because this line `user.contributor.admin || (!msg.sent && user.flags.communityGuidelinesAccepted)` is supposed to prevent you from flagging your own message. I'm fine allow for users to report their own message, but maybe there is a better flow rather than flagging? Either way, I just need to make sure everyone else knows.